presto-parser/src/main/java/com/facebook/presto/sql/tree/ExistsPredicate.java

```diff
@@ -21,26 +21,26 @@
 public class ExistsPredicate
         extends Expression
 {
-    private final Query subquery;
```

Comment:
Sorry, what I meant is change it to Expression and use a SubqueryExpression for the Query case